RoosterPlanner.Api/Controllers/PersonsController.cs

```diff
@@ -35,22 +36,24 @@ public PersonsController(IMapper mapper, IPersonService personService, IProjectS
         [HttpGet("{id}")]
         public async Task<ActionResult> Get(Guid id)
         {
-            PersonViewModel personVm = new PersonViewModel();
+            PersonViewModel personVm = null;
 
             try
             {
-                TaskResult<Person> result = null;//await this.personService.Get(id);
+                TaskResult<User> result = await this.personService.GetUser(id);
                 if (result.Succeeded)
```

Comment:
Zeker in Controller methoden checken of de 'if' geen else moet hebben. Indien niet 'Succeeded' een Http BadRequest of iets dergelijks terug geven. Als de result nu niet 'Succeeded' is, wordt er een 'null' terug gestuurd. Dan misschien beter om object 'personVm' wel te instantieren ipv aan null gelijk te stellen.